test/Confluent.Kafka.IntegrationTests/Tests/AdminClient_DeleteConsumerGroupOffsets.cs

```diff
@@ -0,0 +1,123 @@
+
+                var committedOffsets = consumer1.Committed(TimeSpan.FromSeconds(10));
+                Assert.Equal(1, committedOffsets.Count);
+                Assert.Equal(offsetToCommit, committedOffsets[0].Offset);
+
+                List<TopicPartition> topicPartitionToReset = new List<TopicPartition>() { new TopicPartition(topic1.Name, 0) };
+                var res = adminClient.DeleteConsumerGroupOffsetsAsync(groupId1, topicPartitionToReset).Result;
+                Assert.Equal(groupId1, res.Group);
+                Assert.Equal(1, res.Partitions.Count);
+                Assert.Equal(0, res.Partitions[0].Partition.Value);
+
+                committedOffsets = consumer1.Committed(TimeSpan.FromSeconds(1));
+                Assert.Equal(1, committedOffsets.Count);
+                Assert.Equal(Offset.Unset, committedOffsets[0].Offset);
+
+                // Ensure consumer is actively subscribed to the topic
+                assignmentDone = false;
+                consumer1.Subscribe(new List<String>() { topic1.Name });
+                while (!assignmentDone)
+                {
+                    // To handle the rebalance
+                    consumer1.Consume(TimeSpan.FromSeconds(1));
+                }
+                consumer1.Commit(new List<TopicPartitionOffset>() { new TopicPartitionOffset(topic1.Name, 0, offsetToCommit) }); // commit some offset for consumer
+
+                committedOffsets = consumer1.Committed(TimeSpan.FromSeconds(10));
+                Assert.Equal(1, committedOffsets.Count);
+                Assert.Equal(offsetToCommit, committedOffsets[0].Offset);
+
+                topicPartitionToReset = new List<TopicPartition>() { new TopicPartition(topic1.Name, 0) };
+                try
+                {
+                    res = adminClient.DeleteConsumerGroupOffsetsAsync(groupId1, topicPartitionToReset).Result;
+                    Assert.True(false); // expecting exception.
+                }
+                catch (AggregateException ex)
+                {
+                    var dcgoe = (DeleteConsumerGroupOffsetsException)ex.InnerException;
+                    Assert.Equal(groupId1, dcgoe.Result.Group);
+                    Assert.Equal(1, dcgoe.Result.Partitions.Count);
+                    Assert.Equal(0, dcgoe.Result.Partitions[0].Partition.Value);
+                }
+
+                committedOffsets = consumer1.Committed(TimeSpan.FromSeconds(1));
+                Assert.Equal(1, committedOffsets.Count);
+                Assert.Equal(offsetToCommit, committedOffsets[0].Offset); // offset is unchanged as the consumer is actively subscribed to the topic
+
+                consumer1.Unsubscribe();
+
+                // Deleting offset from the topic which has no set offset
+                consumer1.Assign(new List<TopicPartition>() { new TopicPartition(topic2.Name, 0) });
```

Comment:
i think remove this to test that it works even if not assigned (which it should)

Reply:
Tried removing this, but not working. I am assigning `topic2` here to the consumer but without this call still the previous topic `topic1` is assigned.